services/visual-studio-app-center/visual-studio-app-center-base.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+
+const keywords = [
+  'visual-studio',
+  'vsac',
+  'visual-studio-app-center',
+  'app-center',
+]
+
+const documentation =
+  "You will need to create a <b>read-only</b> API token <a target='_blank' href='https://appcenter.ms/settings/apitokens'>here</a>."
+
+class BaseVisualStudioAppCenterService extends BaseJsonService {
+  async fetch({
+    owner,
+    app,
+    token,
+    schema,
+    url = `https://api.appcenter.ms/v0.1/apps/${owner}/${app}/releases/latest`,
+  }) {
+    return this._requestJson({
+      schema,
+      options: {
+        headers: {
+          'X-API-Token': token,
+        },
+      },
+      errorMessages: {
+        401: 'invalid token',
+        403: 'project not found',
```

Comment:
I'm not familiar with the API here, but are they really returning 403 and 404 for project not found scenarios? I would've guessed 403 would be some kind of forbidden/trying to access a project the token is not authorized to access

Reply:
Surprisingly, yes.
401s are always returned for invalid API tokens and 403/404s for invalid projects.

Interestingly, very occasionally the 404 message will display 'user not found' but this is not consistent so I did not include it.

Reply:
Fun!